dtcli/scripts/dt.py

```diff
@@ -677,6 +677,34 @@ def delete(**kwargs):
     delete_extension.wipe(fqdn=kwargs["extension"], tenant=kwargs["tenant_url"], token=token)
 
 
+@extension.command(
+    help="Validate extension with schemas"
+)
+@click.option(
+    "--instance",
+    default=EXTENSION_YAML, show_default=True,
+    help="Extension file",
+)
+@click.option(
+    "--schema-entrypoint",
+    default=SCHEMAS_ENTRYPOINT, show_default=True,
+    help="Directory with extension schemas",
+)
+def validate_schema(instance, **kwargs):
+    errors = vs.validate_schema(instance_file=instance, schema_entrypoint=kwargs["schema_entrypoint"])
+    invalid = False
+    if errors:
+        invalid = True
+        for i,e in enumerate(errors):

```

Comment:
This is all fine and dandy, but maybe there is a better way of displaying this info. Probably we can create a ticket for that and engage someone for UX

Reply:
Created issue https://github.com/dynatrace-oss/dt-cli/issues/92